packages/firestore/test/util/equality_matcher.ts

```diff
@@ -69,10 +69,13 @@ export function addEqualityMatcher(): void {
         return function(...args): void {
           if (isActive) {
             const [right, msg] = args;
-            utils.flag(this, 'message', msg);
             const left = utils.flag(this, 'object');
 
-            new chai.Assertion(left).assert(
+            // These are used by chai to log the relevant stack trace.
```

Comment:
Would using this make more sense: https://www.chaijs.com/api/plugins/#method_transferflags ?

Optional.

Reply:
Good idea, done.  While testing this I also noticed that the actual/expected was backwards in the error message due to confusing chai docs, so I fixed that as well.